DataFormats/FEDRawData/src/classes_def.xml

```diff
@@ -24,4 +27,5 @@
  </class>
  <class name="edm::Wrapper<FEDRawDataCollection>" splitLevel="0"/>
  <class name="edm::RefProd<FEDRawDataCollection>"/>
+ <class name="edm::Wrapper<BSTRecord>" splitLevel="0"/>
```

Comment:
@slava77 I think you are right. The [template class instance is saved as a private data member](https://github.com/cms-sw/cmssw/blob/master/DataFormats/Common/interface/Wrapper.h#L74) inside `edm::Wrapper`, so I can argue the no-branch-splitting applies also to the `BSTRecord` instance.